docs/protos.md

```diff
@@ -1,33 +1,40 @@
-# Statement protobuf definitions
+# Spec protobuf definitions
 
-Protobuf definitions for the in-toto statement type and some predicates are provided in the
-spec/ directory.  Pre-generated Go implementation of those protos are available in the go dir.
+Protobuf definitions for the in-toto spec and some predicates are
+provided in the spec/ directory.  Pre-generated Go implementation of those
+protos are available in the go/ directory.
 
-go/example/main.go provides an example of how these protos can be used.
+## Pre-requisites
 
-To try it:
+On an Ubuntu-based system, install the following dependencies.
 
 ```shell
-$ make run
-...
-Read statement with predicateType https://example.com/unknownPred2
-Predicate fields:{key:"foo"  value:{struct_value:{fields:{key:"bar"  value:{string_value:"baz"}}}}}
+sudo apt install protobuf-compiler golang
 ```
 
-Please consider providing a proto version of any new predicates proposed.
-
 ## Regenerating Go proto libraries
 
 [It's typical to keep generated Go code in the repository itself](https://go.dev/doc/articles/go_command#:~:text=and%20then%20check%20those%20generated%20source%20files%20into%20your%20repository)
-since it makes users lives much easier.
+since it makes users' lives much easier.
 
-Proto libraries should be regenerated & commited after any change to the proto files:
+Proto libraries should be regenerated & commited after any change to the
+proto files:
 
 ```shell
 $ make go_protos
-go get google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1
-protoc --go_out=go --go_opt=paths=source_relative ./spec/predicates/vsa.proto ./spec/v1.0/statement.proto
 $ git commit -asm "update protos"
-[statement_proto 5edb2c6] Update protos
 ...
 ```
+
+## Run the Go example
+
+go/example/main.go provides an example of how these protos can be used.
+
+To try it:
+
+```shell
+$ make run
+...
+Read statement with predicateType https://example.com/unknownPred2
+Predicate fields:{key:"foo"  value:{struct_value:{fields:{key:"bar"  value:{string_value:"baz"}}}}}
+```
```

Comment:
nit: should there be a newline at the end of this?  Is there already and GitHub is hiding it?

Reply:
IIRC github highlights the absence of a newline with a red minus in a circle or something like that? It's a little annoying because this view is not in fact the source view after all..

Reply:
![image](https://user-images.githubusercontent.com/8928778/226706093-ce57a1fb-bf4e-4fad-8be9-39d74688b1f2.png)
here's an example